pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -316,20 +322,20 @@ func initHandler() {
 	}
 }
 
-func writeToCachedFile(inter interface{}, fileName, name string) error {
+func writeToCachedFile(inter interface{}, fileName string, uid types.UID, name string) error {
```

Comment:
@phoracek that's a remnant of a previous draft version where we were reusing the same function for writing two cached file types.

Reply:
@phoracek let me understand the root of the concern.

Is it that the launcher has to read from a file name that contains the VMI ID? The file is created by virt-handler and because it handles different VMIs, it needs the ID in the file name to not override a wrong cache file. Before the PR, the file was written by virt-launcher itself, in which case there was no need to be specific about the ID.

Or is the concern about significant portion of the code executed in virt-handler context is physically located under virt-launcher path? This is a very valid concern, and I gave it some thought myself. What I figured is that, considering it's just part of the job and we'll have to move more code from launcher into handler runtime context (=> DHCP), we may be better off holding off code refactoring for now and handle it as the final piece in the puzzle once functionality changes are settled. What do you think?